python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -13,7 +13,7 @@
 from lsst.pex.config import (Config, Field, ConfigField, ListField, DictField, ConfigDictField,
                              ConfigurableField)
 from lsst.pipe.base import CmdLineTask, ArgumentParser, TaskRunner, TaskError
-from lsst.coadd.utils import TractDataIdContainer
+from lsst.pipe.drivers.utils import TractDataIdContainer
```

Comment:
Also update  `ups/pipe_analysis.table` so the package depends on `pipe_drivers` rather than `coadd_utils`?

Reply:
There is another import like this in `colorAnalysis.py` so updating that as well?

Reply:
Thank you for catching those!  I've updated and ran a colorAnalysis to make sure it's still happy.